### PR TITLE
Shortcodes + code overflow CSS fix

### PR DESCRIPTION
**Pull request by dcossyleon**
@apreshill - the exciting part of this PR is the addition of a shortcode easy creation of a div tip "note" box with `{{%  note  %}} my text here {{/% note %}}`. 

image

But you may want to double check that they are working as expected. I had trouble getting the shortcode to respect markdown syntax, even when using the `%` notation.

### CODE REVIEW

File: layouts/shortcodes/note.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<div class="note">{{.Inner}}</div>
```
Comment (apreshill) [inline layouts/shortcodes/note.html]:
```suggestion
<div class="note">{{ .Inner | markdownify }}</div>
```

**Comment by apreshill** [comment]
This is awesome! I wonder if we could add 2 more flavors:

1) an rstudio tip (like https://github.com/hadley/r-pkgs/blob/master/style.css#L30-L32; [see](https://r-pkgs.org/whole-game.html#create_package))

2) maybe a warning? https://www.flaticon.com/free-icon/error_645918?term=error&page=1&position=1

**Comment by dcossyleon** [comment]
Great! Two new flavors added, previewed below

image

**Review by juliasilge — APPROVED** [review]
These look *beautiful*. It looks like bold and italic work, along with backticks?

I'll have to think about how we might use an RStudio tip box (it's less obvious to me compared to creating a package) but the other two will be super helpful.

**Comment by apreshill** [comment]
@juliasilge yes- we made them so all "inner" bits can be written in markdown, just as normal text 🌟

**Pull request merged**